### DISCUSSION TITLE
Getting platform specific access token

### DISCUSSION THREAD

**Discussion by nxy7** [ory/kratos#2470] [Q&A] [answered]
Hello, I'm creating app that intends to make google api calls on behalf of user (in 'offline' mode). To do that I need to obtain user access token, do I have to implement it myself or is there a way to get it from kratos?
I'm mainly asking because according to docs /admin/identities endpoints yields only initial access/refresh tokens, and as far as i understand those will be replaced later.

**Comment by aeneasr** [comment, marked as answer]
You can get it from Ory: https://www.ory.sh/docs/kratos/guides/retrieve-social-sign-in-access-refresh-id-token

**Reply by nxy7** [reply]
What does 'initial' mean then? Will it change when google issues new access token?

**Reply by aeneasr** [reply]
> Please be aware that these tokens are only set when an identity sign ups with, or links a new Social Sign In provider. They'ren't updated when an identity signs in!
